biojava-core/src/test/resources/NM_000266.gb

```diff
@@ -120,7 +120,7 @@ FEATURES             Location/Qualifiers
                      /gene="NDP"
                      /gene_synonym="EVR2; FEVR; ND"
                      /note="Norrie disease (pseudoglioma)"
-                     /db_xref="GeneID:4693"
+                     /db_xref="         :4693"
                      /db_xref="HGNC:7678"
```

Comment:
Hi,

not sure why you are removing the text GeneID here? this is content that is also in the original file:

https://www.ncbi.nlm.nih.gov/nuccore/223972707?report=genbank